google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/EncryptionConfiguration.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2016 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigquery;
+
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import java.util.Objects;
+
+public final class EncryptionConfiguration implements Serializable {
```

Comment:
Consider using `@AutoValue` for this

Reply:
It needs to be parsed and serialized to proto. It is easier for me to follow existing pattern here.

Reply:
OK with me. I suppose it's not hard to change later.